src/config.ts

```diff
@@ -423,7 +431,7 @@ const store = {
     'xboxss',
     'xboxsx',
   ]),
-  stores: envOrArray(process.env.STORES, ['amazon', 'bestbuy']).map(entry => {
+  stores: envOrArray(process.env.STORES, ['amazon', 'bestbuy', 'newegg', 'nvidia-gb', 'memoryexpress', 'canadacomputers']).map(entry => {
```

Comment:
```suggestion
  stores: envOrArray(process.env.STORES, ['amazon', 'bestbuy']).map(entry => {
```

I'd like to keep this slim. It's mostly just for example and to show the app works when first booted.

Reply:
Done